src/runtime/hexagon_host.cpp

```diff
@@ -202,27 +203,51 @@ WEAK int halide_hexagon_initialize_kernels(void *user_context, void **state_ptr,
     // created/inserted here.
     ScopedMutexLock lock(&thread_lock);
 
+    // Initialize the runtime, if necessary.
+    if (!shared_runtime) {
+        debug(user_context) << "    Initializing shared runtime\n";
+        const char soname[] = "libshared_runtime.so";
```

Comment:
would it be good to have a slightly halide centric name for this library  "libshared_runtime.so" ?